### ISSUE TITLE
Spinner buttons invisible

### ISSUE THREAD

**Issue by sjaakp** [yiisoft/yii2-jui#73]
After this update, the up/down buttons of the Spinner widget will be unvisible. The buttons are there and they work, but they're just not visible.

To reproduce, just have a page with a Spinner widget on it, and upgrade yii2-jui to version 2.0.7.

The issue is the result of a subtle conflict between **jquery-ui.js 1.12.x** and **bootstrap.js**. It is explained in detail [here](https://github.com/twbs/bootstrap/issues/6094#issuecomment-11148540).

As a workaround, make JuiAsset load version 1.11.4 of jquery-ui in stead of version 1.12.1 (for instance via the assetManager app-component).

Yii version 2.0.13
PHP version 7.1.7
Windows 10
(but it will apply to other versions as well)

**Comment by samdark** [comment]
1.11.4 doesn't work well with jQuery 3.x.

**Comment by cronfy** [comment]
Same problem. Is there any fix for jquery-ui 1.12.1?

**Comment by cronfy** [comment]
Looks like `noConflict()` from bootstrap resolves.this. After loading `bootstrap.js` I added  these lines (from their [docs](https://getbootstrap.com/docs/3.3/javascript/#js-noconflict)):

```
var bootstrapButton = $.fn.button.noConflict() // return $.fn.button to previously assigned value
$.fn.bootstrapBtn = bootstrapButton            // give $().bootstrapBtn the Bootstrap functionality
```

Now spinner buttons are visible.

**Comment by machour** [comment]
Fixed in #45
